### PR TITLE
Support cascade delete of tenants

### PR DESCRIPTION
**Pull request by aviadl**
This breaks compilation with previous SDK versions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -652,7 +652,8 @@ err := descopeClient.Management.Tenant().CreateWithID(context.Background(), "my-
 err := descopeClient.Management.Tenant().Update(context.Background(), "my-custom-id", "My Tenant", tenantRequest)
 
 // Tenant deletion cannot be undone. Use carefully.
-err := descopeClient.Management.Tenant().Delete(context.Background(), "my-custom-id")
+// Path true to cascade value, in case you want to delete all users/keys associated only with this tenant
```
Comment (dorsha) [inline README.md:655]: Pass

File: descope/sdk/mgmt.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ type Tenant interface {
 	// Delete an existing tenant.
 	//
 	// IMPORTANT: This action is irreversible. Use carefully.
-	Delete(ctx context.Context, id string) error
+	Delete(ctx context.Context, id string, cascade bool) error
```
Comment (dorsha) [inline descope/sdk/mgmt.go:38]: Add here also a comment about the attribute

Comment (dorsha) [reply descope/sdk/mgmt.go:38]: You forgot this.

Comment (aviadl) [reply descope/sdk/mgmt.go:39]: done

**Review by dorsha — APPROVED** [review]

**Pull request merged**